src/renderer/mesh_storage.zig

```diff
@@ -885,6 +885,7 @@ pub const MeshGenerationTask = struct { // MARK: MeshGenerationTask
 		.isStillNeeded = @ptrCast(&isStillNeeded),
 		.run = @ptrCast(&run),
 		.clean = @ptrCast(&clean),
+		.taskType = .lighting,
```

Comment:
I think "meshgenAndLighting" would be more fitting.

Reply:
Is it just MeshGenerationTask that needs to be added to meshgenAndLighting?

Reply:
I would say that no two tasks should have the same type (except from misc), otherwise the information becomes less useful.